### ISSUE TITLE
Fix dependency issue causing IIS build failure

### ISSUE THREAD

**Issue by marekhorst** [openaire/iis#1217]
The `dhp-schemas:1.2.4-dhp_oaf_model`, which is the new version of one of our DNet dependencies, has caused our master branch build to fail:

http://ci.ceon.pl/view/IIS/job/iis_build_multibranch/job/master/

The module was released with `dhp_oaf_model` suffix to indicate this is not a final release but experimental one. Since DNet dependencies are not defined explicitly in IIS but with version ranges this makes this version to be picked up by our master branch.

This is the original redmine thread with reasoning behind DNet module release:
https://issue.openaire.research-infrastructures.eu/issues/6281#note-10

We should:
1) fix IIS build and integration tests since it is broken for more than a week 
2) find a bulletproof solution for handling such cases in future: when some DNet lib is released with suffix in its version our IIS master branch should not pick this up (if we need to rely on this version we can define it explicitly on a branch we are working on). This may require @claudioatzori involvement in agreeing on the way DNet releases are marked when providing experimental feature (using `qualifier` or some other more appropriate approach).

**Comment by przemyslawjacewicz** [comment]
Following our discussion during daily probably the most appropriate solution would be to bump major version of the dependency to 2.* because the changes are breaking. The version range used in IIS, that is `[1.0.0, 2.0.0)` would then guarantee that we pick up the old version and not break the code. We could then proceed with an update 'the usual way', bumping the version and aligning the code on a branch. But bumping the major version is definitely not the way to go because the changes in the dependency can be still be in progress. 

So one way to go is that we could depend on snapshot repositories to avoid any temporary/custom releases. We could enable snapshot repositories in IIS, this would pick up the new version of the dependency, make necessary code updates and wait on a dedicated branch until the final release of the lib. After the release we would disable snapshot repositories, update if necessary the version of the lib and merge into master. This way we could avoid broken builds of IIS, or any other code dependent on the dependency . However, we do not know if this scenario will work, so this would be something to test upon next breaking changes in any dependency.
